client/my-sites/promote-post-i2/components/tsp-banner/index.tsx

```diff
@@ -44,18 +50,17 @@ function TspBanner( props: TspBannerProps ) {
 									'Reach more people and spark conversations by promoting your content as a native Tumblr post, where users can like, reply, and engage directly with your ad.'
 								) }
 								&nbsp;
-								<ExternalLink
-									href={ localizeUrl(
-										'https://wordpress.com/support/promote-a-post/promote-your-content-with-tumblr-native-posts'
-									) }
-									target="_blank"
+								<InlineSupportLink
+									supportContext="blaze_learn_more"
+									showIcon={ false }
 									onClick={ () => {
 										recordTracksEvent( 'calypso_dsp_tsp_banner_learn_more_click', {} );
 									} }
+									// The Help Center doesn't work in Jetpack sites
+									showSupportModal={ isJetpack }
```

Comment:
I think this should be the other way around.
```suggestion
									showSupportModal={ ! isJetpack }
```